cli_service/cli_service-consts.go

```diff
@@ -1,21 +1,22 @@
-// Autogenerated by Thrift Compiler (0.12.0)
-// DO NOT EDIT UNLESS YOU ARE SURE THAT YOU KNOW WHAT YOU ARE DOING
+// Code generated by Thrift Compiler (0.17.0). DO NOT EDIT.
```

Comment:
@mattdeekay why none of the databricks thrift definition optimization (e.g., directResult) is in this PR? 

did we use databricks thrift protocol to update these files or did we use the newer open source thrift protocol to update the thrift definition in this PR?

Reply:
@moderakh the Databricks specific optimisations _are present_ in this updated file. But this PR doesn't include any of the implementations of those interfaces. This PR represents the "lift" portion of "lift and shift" from the OSS Apache Hive Thrift definition file.

> did we use databricks thrift protocol to update these files or did we use the newer open source thrift protocol to update the thrift definition in this PR?

@mattdeekay generated these new stubs using the private Databricks Thrift definition.

Reply:
<img width="1364" alt="Screen Shot 2022-09-22 at 1 49 41 PM" src="https://user-images.githubusercontent.com/22279672/191848918-7578c122-a2e2-4ec3-976b-58ae8365b7f4.png">
I was looking for the changes to `cli_service.go` expecting that to happen in this PR. but github was not showing any changes in that file. (see the screenshot attached). I can see the changes now by going to the specific commit.